lessons/geoparsing-text-with-edinburgh.md

```diff
@@ -0,0 +1,385 @@
+---
+title: Geoparsing Text with the Edinburgh Geoparser
+collection: lessons
+layout: lesson
+slug: geoparsing-text-with-edinburgh
+date: 2017-10-25
+authors:
+- Beatrice Alex
+reviewers:
+- Anouk Lang
+- Sarah Simpkin
+editors:
+- Ian Milligan
+difficulty: 3
+review-ticket: 26
+activity: presenting
+topics: [mapping]
+abstract: "This tutorial teaches users how to use the Edinburgh Geoparser to process a piece of text, extract and resolve the locations contained within it, and plot them as a web map."
+---
+
+{% include toc.html %}
+
+# Introduction
+
+This is a lesson on how to use the [Edinburgh Geoparser](https://www.ltg.ed.ac.uk/software/geoparser/).  The Geoparser allows you to process a piece of text and extract and resolve the locations contained within it. Among other uses, geo-resolution of locations makes it possible to map the data.
+
+The Geoparser works best on running text, as it considers locations in context for disambiguation. For example, if you would like to get a sense of the place names mentioned in a piece of text, the Geoparser can be used to identify terms in a document that are likely to refer to place names.  It will then provide its best guess as to where those places are in terms of latitute/longitude coordinates.
+
+In December 2015, the Edinburgh Geoparser was released under the University of Edinburgh’s GPL license to be used by other researchers in the field of text mining as well as other scholars who are interested in geoparsing text. More information on its documentation, publications about it and how to download it can be found [here](https://www.ltg.ed.ac.uk/software/geoparser/).
+
+A simple online demo of the vanilla Edinburgh Geoparser can be tried out [here](http://jekyll.inf.ed.ac.uk/geoparser.html). It provides only the visual interface to the Geoparser output after uploading a text file and selecting a gazetteer.  The demo is otherwise not configurable and should only be used to try out small examples and not for geo-parsing a large number of files.
+
+The following lesson explains how the Edinburgh Geoparser works under the hood and contains information on:
+
+* Prerequisites and terminology
+*	Downloading and setting up the Edinbugh Geoparser,
+*	Geo-parsing a text file,
+*	Other useful options for running the Geoparser,
+*	Geo-parsing multiple text files, and
+*	Extracting geo-resolution output to TSV.
+
+### Prerequisites and Terminology
```

Comment:
The levels of every header after `Introduction` should be fixed up so they're more logical. Right now, every one is an `h3` or `h4`, and so the generated ToC doesn't end up being very useful. Generally, each section should be at `h2`, with subsections incremented as needed.

Reply:
Thanks @mdlincoln – started this long enough ago I forgot about the ToC. I'll rationalize it.